cloudigrade/util/exceptions.py

```diff
@@ -6,3 +6,27 @@ class InvalidArn(ParseError):
     """an invalid ARN was detected."""
 
     pass
+
+
+class AwsValueError(ValueError):
+    """Inapropriate value given to function."""
+
```

Comment:
You can delete the `pass`es from these exception classes. Having a docstring like this is good enough to keep the structure in place.